core/testing/build.gradle.kts

```
@@ -0,0 +1,19 @@
+plugins {
+    alias(libs.plugins.terning.library)
+}
+
+android {
+    namespace = "com.watcha.testing"
+}
```

패키지 이름 watcha 말구 terning 어때용 
```suggestion
android {
    setNamespace("testing")
}
```
그럼 이렇게 간편하게 나타낼 수도 있답니당

---

헐 이거 뭐야;;;; 쇽샥해왔더니,,,,,,,, 섬세한 발견 감사해용 ㅜㅜ